awx/main/models/credential/__init__.py

```diff
@@ -539,7 +539,7 @@ def build_extra_vars_file(vars, private_dir):
                 f = os.fdopen(handle, 'w')
                 f.write(safe_dump(vars))
                 f.close()
-                os.chmod(path, stat.S_IRUSR)
+                os.chmod(path, stat.S_IRUSR | stat.S_IRGRP)
```

Comment:
I cannot articulate why the extra vars file should not be writable when the others are.